ansible/roles/test/tasks/syslog.yml

```diff
@@ -112,7 +112,6 @@
 
 # Check Messages
 - name: Check syslog messages for the test message
-  connection: local
```

Comment:
The intended behavior here is to test the *remote* syslog behavior, therefore we actually *want* to check the local syslog in this step, to ensure the DUT sends its messages to the remote syslog server, which should be running on the *local* machine.

Reply:
Hi Joe

Thanks for review. In this case, kindly let me know. Do we need to manually start a remote syslog server on Controller or below steps in playbook are sufficient for this.

My guess is : below 2 modules does the job of starting remote syslog server.

- name: Add Rsyslog destination for testing
 39   become: true
 40   lineinfile:
 41     line: "*.* @{{ local_srcip }}:{{ syslog_port }}"
 42     dest: /etc/rsyslog.conf
 43     state: present
 44 
 45 - name: Restart Syslog Daemon
 46   become: true
 47   service: name=rsyslog state=restarted

-- I do not see syslogs coming to controller on my setup.

Reply:
No, those lines configure rsyslog running on the DUT to be aware of the syslog server it should send to for the test. However, lines 51-95 should start the remote syslog server on the Ansible host.